pkg/controllers/raycluster_controller.go

```diff
@@ -350,5 +354,19 @@ func (r *RayClusterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		Named(controllerName).
 		For(&rayv1.RayCluster{}).
+		Owns(&corev1.ServiceAccount{}).
+		Owns(&corev1.Service{}).
+		Owns(&corev1.Secret{}).
+		Owns(&routev1.Route{}).
+		Owns(&networkingv1.Ingress{}).
+		Watches(&rbacv1.ClusterRoleBinding{}, handler.EnqueueRequestsFromMapFunc(
+			func(c context.Context, o client.Object) []reconcile.Request {
+				return []reconcile.Request{{
+					NamespacedName: client.ObjectKey{
+						Name:      o.GetLabels()["ray.io/cluster-name"],
```

Comment:
Should we check for the absence of these labels? The watch will receive events for all CRBs. An alternative is to configure the cache to filter events only for our CRBs.

Reply:
I've changed it to check for the absence of the labels, I'm not sure how to configure the cache to filter for only our CRBs

Reply:
I've created https://github.com/project-codeflare/codeflare-operator/tree/pr-cache-example that shows how to configure the manager cache.

Reply:
Thanks Antonin, I'm adding this too so that it can be followed if someone finds this
https://github.com/kubernetes-sigs/controller-runtime/blob/main/designs/use-selectors-at-cache.md

Reply:
For the record, this broke the AppWrapper controller.   I submitted a fix here #640